src/collective/cover/tests/cover.robot

```diff
@@ -44,7 +44,8 @@ Setup Cover Test Case
 Click Add Cover
     Open Add New Menu
     Click Link  css=a#collective-cover-content
-    Page Should Contain  Add Cover
+    Wait Until Page Contains  Add Cover
+    Sleep  1s  Wait for js
```

Comment:
@hvelarde Why don't we use this instead? https://github.com/collective/collective.cover/blob/master/src/collective/cover/tests/test_calendar_tile.robot#L46